pkg/kubelet/pod_workers.go

```diff
@@ -181,7 +181,7 @@ func (p *podWorkers) managePodLoop(podUpdates <-chan UpdatePodOptions) {
 			update.OnCompleteFunc(err)
 		}
 		if err != nil {
-			glog.Errorf("Error syncing pod %s, skipping: %v", update.Pod.UID, err)
+			glog.Errorf("Error syncing pod %s (%s/%s), skipping: %v", update.Pod.UID, update.Pod.Namespace, update.Pod.Name, err)
```

Comment:
this should use https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/util/format/pod.go#L32